attic/Processes/eclipse.adoc

```diff
@@ -1,6 +1,6 @@
 = Eclipse Foundation
 
-In December 2016, the MicroProfile community was accepted into the Eclipse Foundation as the link:https://projects.eclipse.org/projects/technology.microprofile[Eclipse MicroProfile project]. 
+In December 2016, the MicroProfile community was accepted into the Eclipse Foundation as the link:https://projects.eclipse.org/projects/technology.microprofile[MicroProfile project]. 
```

Comment:
This reference, we shouldn't change.  In 2016, we were accepted and known as Eclipse MicroProfile.  The Eclipse prefix was a requirement.  Maybe we need another line about becoming a Working Group as of such-and-such date and now we are known as MicroProfile.

Reply:
It needs to change to MP alone. The year is 2017 formally, not 2016. :)

Reply:
Fixed